### PR TITLE
rework the script connection #283

### PR DESCRIPTION
**Pull request by polina-belyakaeva**
Added a timestamp query parameter to the script URL to force browsers to load the latest version of the script

### CODE REVIEW

**Comment by fey** [comment]
@polina-belyakaeva Hi, can you add to JS widget console log that print something? For example current commit hash in main branch? You can hardcode hash. We need id to test loading new version of JS widget

**Comment by polina-belyakaeva** [comment]
@fey I added console log, could you check it, please?

**Comment by fey** [comment]
@polina-belyakaeva please add console.log to the widget body itself. After all, we are checking if a new version of the script is loaded https://github.com/Hexlet/hexlet-correction/blob/main/src/widget/index.js

**Comment by fey** [comment]
@polina-belyakaeva too many changes =) Please undo commit (without creating new commit) and remove changes with quotes (please check your editor settings).
If need help - i am here)

File: src/widget/index.html [hunk 1]
```diff
@@ -217,10 +217,16 @@ <h2 class="text-body-emphasis">Guides</h2>
...
+            const script = document.createElement('script');
+            script.src = 'https://cdn.jsdelivr.net/gh/hexlet/hexlet-correction@main/src/widget/index.js?v=' + new Date().getTime();
+            document.body.appendChild(script);
+        });
+    </script>;
     <script>
         handleTypoReporter({ authorizationToken: 'MjQyOmVlYTZkMWE1LTNkMGQtNDg1Yi04OGMwLWVkOGU1YTRlOGZjMA==',
         workSpaceUrl: 'https://hexlet-correction.herokuapp.com', workSpaceId: '242'})
-    </script>
+    </script>
```
Comment (fey) [inline src/widget/index.html:230]: @polina-belyakaeva whitespace

File: src/widget/index.js [hunk 2]
```diff
@@ -1,5 +1,7 @@
+console.log("d495bd24a1e9db08f68b63d5fbc238c4e143931c");
+
 const generateModalStyles = () => {
-  const modalStyleEl = document.createElement('style');
+  const modalStyleEl = document.createElement("style");
```
Comment (fey) [inline src/widget/index.js:4]: @polina-belyakaeva your commit history still have commit with changed quotes

**Comment by fey** [comment]
@polina-belyakaeva merged, but not works =)
Exaplained in mattermost chat =)

**Pull request merged**